config/initializers/devise.rb

```diff
@@ -232,7 +232,7 @@
   # Turn scoped views on. Before rendering "sessions/new", it will first check for
   # "users/sessions/new". It's turned off by default because it's slower if you
   # are using only default views.
-  # config.scoped_views = false
+  config.scoped_views = true
```

Comment:
can you explain what is happening here?

Reply:
We had to change this to visually see the new name field on login.
Link - https://gist.github.com/withoutwax/46a05861aa4750384df971b641170407